cppcache/integration/test/PartitionRegionOpsTest.cpp

```diff
@@ -144,6 +146,10 @@ void verifyMetadataWasRemovedAtFirstError() {
       }
     }
   }
+  std::cout << "timeoutErrors: " << timeoutErrors << ", ioErrors: " << ioErrors
+            << ", metadataRemovedDueToTimeout: " << metadataRemovedDueToTimeout
+            << ", metadataRemovedDueToIoErr: " << metadataRemovedDueToIoErr
+            << std::endl;
```

Comment:
If you really want to see this output when running your test, I believe it's best to use std::cerr rather than cout.  This looks like leftover debugging trace stuff to me, tho.

Reply:
Tests should never output anything other than assertion failures. If more information is useful to log then it is useful to assert.